packages/wcg-url/common.ts

```diff
@@ -0,0 +1,3 @@
+import { Observable } from '@nativescript/core';
+
+export class WcgUrlCommon extends Observable {}
```

Comment:
Would strongly prefer this (and all other polyfills) weren't coupled to Core at all, so that we could use it for Coreless NativeScript. Don't see much benefit from extending Observable in any case, as it adds reactive functionality the specs don't require.